app/views/usage/index.html.erb

```diff
@@ -1,8 +1,24 @@
 <% title _('Usage statistics') %>
 
 <div class="row">
-  <div class="col-md-12">
+  <div class="col-md-12" >
     <h1>Usage statistics</h1>
+  </div>
+</div>
+<div class="row" style="margin: 15px 0;">
```

Comment:
Can we avoid adding styles in the HTML. 

Please add CSS classes or use existing CSS for these.